src/bsaextractor.cpp

```diff
@@ -91,7 +91,7 @@ void BsaExtractor::modInstalledHandler(const QString &modName/*, EFileCategory i
 
   QFileInfoList archives = dir.entryInfoList(QStringList("*.bsa"));
   if (archives.length() != 0 &&
-      (QuestionBoxMemory::query(nullptr, "unpackBSA", tr("Extract BSA"),
+      (QuestionBoxMemory::query(nullptr, "unpackBSA", tr("Extract BSA"), modName,

```

Comment:
This appears unnecessary, I'd prefer if the old api for QuestionBoxMemory without a "filename" parameter was still an option and in this case, the "remember for" checkbox was hidden.